bin/varnishd/cache/cache_req_fsm.c

```diff
@@ -407,12 +404,10 @@ cnt_transmit(struct worker *wrk, struct req *req)
 			    "Content-Length: %jd", req->resp_len);
 	}
 
-	if (err == 0)
```

Comment:
see previous comment, I don't think we should push error reporting down into VDP

Reply:
See comments on the error handling above.